aip/general/0121.md

```diff
@@ -61,6 +61,10 @@ exposes a large number of resources with a small number of methods on each
 resource. The methods can be either the standard methods ([Get][], [List][],
 [Create][], [Update][], [Delete][]), or [custom methods][].
 
+A resource that supports a [Create][], [Update][], or [Delete][] must also
+support [Get][] to enable the user to validate the success of their operation
+outside of the operation completion.
```

Comment:
I'm going to use "validate the state of the resource", since that's actually what's being done here. Not so much "operation completion", which should be detected via LRO or request completion